.circleci/config.yml

```diff
@@ -76,6 +85,29 @@ commands:
             - llvm-project/clang/include
             - llvm-project/lld/include
             - llvm-project/llvm/include
+  hack-ninja-jobs:
+    steps:
+      - run:
+          name: "Hack Ninja to use less jobs"
+          command: |
+            echo -e '#!/bin/sh\n/usr/bin/ninja -j3 "$@"' > /go/bin/ninja
+            chmod +x /go/bin/ninja
+  build-binaryen:
```

Comment:
Suggestion: this might be clearer when the name is `build-binaryen-linux`. Because the cache is only for Linux.
Similarly, I think it's better to rename the cache key to something like `binaryen-linux-v1`.